content/guides/asset-management.md

```diff
@@ -126,41 +382,126 @@ npm install --save-dev csv-loader xml-loader
 
 __webpack.config.js__
 
-``` js
-module.exports = {
-  //...
-  module: {
-    rules: [
-      {
-        test: /\.(csv|tsv)$/,
-        use: 'csv-loader'
-      },
-      {
-        test: /\.xml$/,
-        use: 'xml-loader',
-      }
-    ]
-  },
-  //...
-}
+``` diff
+  var path = require('path');
+
+  module.exports = {
+    entry: './src/index.js',
+    output: {
+      filename: 'bundle.js',
+      path: path.resolve(__dirname, 'dist')
+    },
+    module: {
+      rules: [
+        {
+          test: /\.css$/,
+          use: [
+            'style-loader',
+            'css-loader'
+          ]
+        },
+        {
+          test: /\.(png|svg|jpg|gif)$/,
+          use: [
+            'file-loader'
+          ]
+        },
+        {
+          test: /\.(woff|woff2|eot|ttf|otf)$/,
+          use: [
+            'file-loader'
+          ]
+        },
++       {
++         test: /\.(csv|tsv)$/,
++         use: [
++           'csv-loader'
++         ]
++       },
++       {
++         test: /\.xml$/,
++         use: [
++           'xml-loader'
++         ]
++       }
+      ]
+    }
+  };
+
+```
+
+Add some data files to your project:
+
+__project__
+
+``` diff
+  webpack-demo
+  |- package.json
+  |- webpack.config.js
+  |- /dist
+    |- bundle.js
+    |- index.html
+  |- /src
++   |- data.xml 
+    |- my-font.woff
+    |- my-font.woff2
+    |- icon.png
+    |- style.css
+    |- index.js
+  |- /node_modules
+```
+
+__src/data.xml__
+
+``` xml
+<?xml version="1.0" encoding="UTF-8"?>
+<note>
+  <to>Mary</to>
+  <from>John</from>
+  <heading>Reminder</heading>
+  <body>Call Cindy on Tuesday</body>
+</note>
 ```
 
-Now you can `import` any one of those four types of data (JSON, CSV, TSV, XML) and the `Data` variable you import it to will contain parsed JSON for easy consumption:
+Now you can `import` any one of those four types of data (JSON, CSV, TSV, XML) and the `MyData` variable you import it to will contain parsed JSON for easy consumption:
```

Comment:
This doesn't correspond with the line above where `Data` is used as the variable... I would either drop this change or change them both to `MyData`.